container/features/src/main/resources/features.xml

```diff
@@ -357,12 +357,13 @@
       <bundle>wrap:mvn:org.springframework.webflow/spring-webflow/${springWebFlowVersion}</bundle>
     </feature>
 
-    <feature name="jicmp" description="jicmp" version="1.4.4">
-      <bundle>wrap:mvn:org.opennms/jicmp-api/1.4.4$Export-Package=org.opennms.protocols.icmp,org.opennms.protocols.ip</bundle>
+    <feature name="jicmp" description="jicmp" version="2.0.0">
+      <feature>guava</feature>
+      <bundle>wrap:mvn:org.opennms/jicmp-api/2.0.0$Export-Package=org.opennms.protocols.icmp,org.opennms.protocols.ip</bundle>
```

Comment:
Can we change these JARs into bundle artifacts so that we don't have to use the ```wrap:``` protocol any more? That would be nice.